functions/ts/kubeval/src/kubeval.ts

```diff
@@ -158,12 +132,6 @@ kubeval.usage = `
 Validates configuration using kubeval.
 
 Configured using a ConfigMap with the following keys, all of which are optional:
-${SCHEMA_LOCATION}: The base URL used to download schemas.  If not specified,
```

Comment:
This change is backwards incompatible. Can we retain these flags but not use them in default declarative flow? In imperative flow, if the user really wants to mount additional schema and use these additional flags, then that's OK. Folks from Spotify have added these, so we should avoid breaking that flow if possible.

Reply:
The eventual workflow for fn kubeval is like:
1. openapi doc is injected by kpt at some location. (we will bake a openapi doc in the image in the near term)
2. We convert the openapi to json schema and write it to `SCHEMA_LOCATION`.
3. kubeval validates resources against the json schema in `SCHEMA_LOCATION`

However, kpt fn execute functions as `nobody`, if `SCHEMA_LOCATION` is not `/tmp`, we cannot write the json schema files in step 2.

Reply:
Thinking a bit more about it, it seems what we can do is:
If users specify `SCHEMA_LOCATION`, we always use that.
If `SCHEMA_LOCATION` is not specified, we will use `/tmp` as the default and use the baked-in openapi (or the one injected by kpt in the future).

Reply:
SGTM